api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -149,6 +149,14 @@ message RateLimit {
   // the fraction of requests to enforce rate limits on. And the default percentage of the
   // runtime key is 100% for backwards compatibility.
   config.core.v3.RuntimeFractionalPercent filter_enforced = 15;
+
+  // When set to true, the filter will emit per-stream stats for access logging. These stats will be stored in the
+  // filter state under the filter name. The filter will emit latency, bytes sent/received, upstream host, and
+  // upstream cluster info.
+  //
+  // .. note::
+  //   Stats are only emitted to the filter state if a check request is actually made to the rate limit service.
+  bool emit_filter_state_stats = 16;
```

Comment:
If this doesn't change any behavior and won't bring too much overhead, then I think we can enable it by default and needn't additional flag.